assets/app/view/game/corporation.rb

```diff
@@ -48,8 +48,10 @@ def render
           children << h(Companies, owner: @corporation, game: @game) if @corporation.companies.any?
         end
 
-        abilities = @corporation.all_abilities.select { |ability| ability.owner.corporation? }
-        children << render_abilities(abilities) if abilities&.any?
+        abilities_to_display = @corporation.all_abilities.select do |ability|
+          ability.owner.corporation? && !ability.name.nil?
+        end
+        children << render_abilities(abilities_to_display) if abilities_to_display&.any?
```

Comment:
abilities_to_display will never be nil no need for &

Reply:
Fixed

Reply:
Fixed